royalties/contracts/LibRoyalties2981.sol

```diff
@@ -17,7 +17,7 @@ library LibRoyalties2981 {
         if (amount == 0) {
             return result;
         }
-        uint256 percent = (amount * 100 / _WEIGHT_VALUE) * 100;
+        uint256 percent = amount * 10000 / _WEIGHT_VALUE;
         require(percent < 10000, "Royalties 2981, than 100%");
```

Comment:
maybe better phrase in require "Royalties 2981, than 100%" change to "Royalties 2981 exceeds 100%" ?

Reply:
sure, fixed